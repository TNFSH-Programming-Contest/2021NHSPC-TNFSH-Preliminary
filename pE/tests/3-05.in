500 1000
3 4 7
5 3 17
5 4 7
6 4 23
7 8 23
6 5 7
8 9 20
10 7 23
6 4 18
6 11 6
6 11 8
6 4 17
4 11 14
12 13 7
13 10 28
13 10 29
13 9 12
6 11 6
13 9 16
14 12 7
15 16 5
12 9 28
6 11 18
12 9 14
14 9 18
11 17 20
15 16 25
18 9 27
19 17 18
18 14 27
20 21 9
21 17 18
16 19 18
15 17 24
20 6 9
22 23 7
9 15 11
20 17 20
17 24 15
22 23 8
24 25 14
20 17 10
20 18 9
26 27 6
25 22 22
28 29 15
23 27 13
23 27 21
23 27 10
28 29 25
28 29 22
28 26 10
26 30 29
26 30 19
28 29 24
28 29 12
23 27 20
26 27 9
30 27 25
29 31 19
26 27 11
23 27 21
23 27 12
31 20 17
20 23 20
28 27 25
28 27 12
32 33 20
34 35 14
27 32 11
27 32 27
33 36 14
36 37 24
38 34 9
38 34 19
34 35 8
34 35 22
34 35 17
34 35 11
39 40 21
40 38 19
28 39 25
28 37 8
37 35 14
28 35 14
28 35 28
41 35 6
42 43 5
28 35 28
28 41 21
44 45 29
42 43 23
41 35 7
28 35 18
46 42 11
45 47 10
46 44 24
28 35 8
48 35 29
28 48 25
47 49 17
47 49 19
50 51 10
28 35 17
44 47 25
44 47 13
50 51 19
50 51 23
52 43 21
49 43 21
46 43 5
53 52 13
43 51 19
46 43 22
46 50 8
28 35 21
54 55 17
46 43 12
46 49 10
49 43 16
46 53 22
51 28 5
43 51 24
46 43 25
54 55 27
56 54 7
46 51 22
46 56 22
46 55 10
55 43 13
57 58 18
46 51 17
59 60 27
35 57 8
61 62 26
63 61 14
46 58 16
61 64 11
46 58 9
64 59 21
65 66 23
58 63 21
67 68 9
66 69 7
46 58 19
62 64 16
62 64 6
63 61 17
58 61 19
58 61 12
70 71 24
70 71 6
72 73 14
73 74 13
65 69 25
75 65 28
70 71 12
69 76 26
74 77 7
78 79 16
80 75 19
79 81 24
77 82 28
83 84 15
74 82 20
81 85 25
86 87 28
68 72 18
71 88 27
60 70 20
85 89 15
90 78 29
86 87 26
67 82 17
91 71 15
92 83 23
67 82 5
93 94 25
89 86 13
46 71 20
90 89 21
76 92 24
93 94 10
46 91 24
67 82 16
88 95 11
96 97 13
80 84 12
95 98 19
82 46 24
95 80 8
90 89 11
99 100 14
98 84 5
87 101 29
102 100 6
103 100 24
93 94 22
67 95 20
99 100 13
99 102 29
90 104 5
99 100 9
95 84 17
97 105 15
90 101 17
101 106 23
90 106 7
104 106 20
105 107 22
99 103 18
108 109 18
93 94 30
93 94 17
84 110 14
93 94 19
111 112 29
112 113 25
94 114 23
111 113 16
115 99 12
90 106 5
67 95 11
116 117 22
118 117 14
118 117 27
118 116 26
108 109 15
109 119 29
120 121 10
110 122 11
123 124 11
119 125 12
100 90 23
126 127 24
117 128 27
129 130 9
131 132 8
127 133 11
134 135 22
131 136 28
136 137 9
131 136 23
121 138 22
137 132 25
125 139 13
140 141 13
129 130 6
142 134 23
93 114 29
143 96 16
111 113 15
106 144 28
124 108 23
145 146 15
115 144 22
131 132 18
111 141 22
147 148 22
120 138 18
120 138 16
149 146 21
126 133 12
142 134 6
149 146 18
148 93 11
107 150 23
151 140 18
152 153 14
152 153 17
149 145 20
111 141 20
129 154 15
114 155 14
156 157 12
128 158 16
133 152 10
159 158 26
143 150 8
131 132 30
159 158 22
160 161 11
156 157 15
162 163 17
159 158 30
154 164 26
155 149 28
165 162 11
157 166 11
140 141 10
131 167 7
168 142 14
118 128 26
152 123 26
139 153 11
169 170 18
131 132 16
171 172 10
163 173 15
115 144 11
174 159 28
175 141 25
174 159 7
153 176 15
172 177 11
118 158 30
178 120 24
151 140 20
130 164 25
179 180 22
153 176 20
162 181 6
158 182 11
183 126 18
157 166 26
180 184 28
185 181 12
135 186 6
113 175 10
158 187 19
188 147 28
138 171 13
189 184 6
173 185 14
122 151 27
159 118 7
174 159 11
158 182 23
144 156 22
161 190 13
168 186 9
191 166 9
174 164 7
185 181 22
167 132 24
177 192 16
111 67 26
193 194 25
195 188 30
179 196 12
191 166 25
170 197 17
191 166 21
174 129 12
198 193 11
158 187 18
164 158 18
199 146 11
190 200 9
161 190 22
201 189 11
202 203 10
189 184 5
197 165 15
179 196 19
204 169 24
205 206 23
191 166 22
207 203 6
171 192 10
198 208 7
131 146 14
208 194 25
209 202 30
210 211 20
212 174 30
183 176 5
209 203 19
161 190 20
160 213 28
191 115 24
171 192 7
214 143 25
186 166 24
215 174 14
158 187 10
212 215 15
216 217 28
210 211 23
201 184 27
179 184 12
218 219 28
209 207 7
213 190 24
179 220 16
198 194 28
201 184 10
141 195 23
174 158 16
221 199 17
222 211 26
209 203 12
196 220 22
150 194 11
192 223 11
220 201 18
132 111 21
210 222 14
160 161 27
158 182 13
205 206 7
141 149 9
141 149 9
214 194 5
212 174 8
219 169 9
169 162 5
209 203 23
224 225 16
141 149 15
200 224 30
149 221 20
191 226 14
158 187 26
182 187 6
158 227 22
179 184 18
179 184 25
223 228 29
183 176 15
228 229 23
211 183 22
214 194 9
218 204 21
226 168 26
230 218 9
214 198 30
231 214 8
214 232 6
212 158 16
168 166 20
212 158 6
181 194 24
227 231 18
187 227 24
233 234 30
216 217 24
235 194 27
166 131 13
212 158 22
236 210 29
146 237 27
179 184 25
238 198 15
198 230 14
227 214 14
179 184 7
239 240 17
209 203 8
232 238 29
241 225 6
242 243 9
244 241 24
178 229 5
245 212 12
237 246 15
158 214 15
241 160 22
239 240 15
247 248 24
249 250 20
251 244 14
251 244 26
252 253 16
233 234 10
179 184 30
206 235 19
252 254 18
255 242 27
252 254 15
214 205 11
236 176 23
256 257 16
235 194 7
258 259 29
260 251 14
258 259 28
261 262 6
253 254 26
263 264 21
217 260 15
258 259 23
176 2 19
258 259 18
250 265 29
179 184 8
252 254 17
245 266 7
229 267 8
268 194 19
203 236 12
269 239 15
268 194 15
176 2 12
270 248 11
271 268 11
245 212 30
256 257 7
258 259 16
264 261 26
272 273 20
257 249 21
203 176 23
274 275 23
263 264 13
216 260 30
263 264 10
258 259 22
270 248 10
276 277 6
278 269 13
246 194 8
279 280 20
269 240 25
264 261 22
271 191 16
267 233 6
281 282 16
283 284 21
283 284 23
285 286 18
286 287 8
244 225 28
272 278 27
250 245 12
266 214 9
288 274 29
265 245 10
245 271 28
209 176 6
245 214 18
289 290 17
255 243 17
291 248 25
292 293 9
255 288 25
287 294 16
276 277 14
295 296 11
297 2 24
298 299 30
255 275 5
271 194 30
243 300 12
225 263 27
261 262 9
301 247 28
294 302 30
179 184 8
298 299 25
293 258 16
234 303 28
304 305 24
300 302 15
301 270 15
306 307 9
308 284 27
295 296 18
303 309 25
273 240 14
257 245 24
310 279 5
301 291 29
295 296 11
283 308 21
311 312 25
313 314 26
307 315 28
304 305 16
289 290 18
316 2 5
290 306 22
317 262 25
318 319 26
316 2 8
216 320 15
277 319 5
280 321 9
312 322 24
323 179 13
257 245 7
318 319 17
324 325 17
304 305 21
320 317 22
216 262 17
326 327 14
328 318 24
179 184 17
322 329 16
330 331 18
328 318 8
301 248 28
323 184 25
332 313 14
326 327 21
333 334 24
315 328 26
335 322 14
334 336 28
337 281 23
272 338 17
240 338 29
259 339 29
340 341 20
300 285 20
216 262 23
341 342 11
234 309 28
301 343 12
335 311 17
295 296 7
338 344 12
345 346 27
281 282 13
347 348 28
349 350 26
295 296 22
290 330 29
283 351 6
234 309 28
326 327 25
352 353 7
354 248 28
282 355 13
323 356 28
349 298 11
357 358 17
292 259 29
277 319 7
346 359 17
357 360 12
302 275 18
346 359 23
326 327 22
262 289 12
255 275 8
331 315 22
301 361 27
362 2 30
329 349 16
357 340 16
301 248 28
318 276 13
342 360 20
299 363 6
364 248 7
313 314 21
351 284 6
363 350 29
351 284 27
296 248 29
347 348 18
255 275 27
301 364 10
344 365 30
349 363 28
366 367 22
362 209 6
252 254 20
216 368 28
366 367 8
369 370 15
352 304 30
368 319 5
304 305 10
184 356 25
366 371 19
358 372 30
352 373 26
374 375 30
335 322 19
376 362 14
256 194 8
255 377 14
310 321 25
343 295 21
357 360 26
255 333 9
376 2 12
370 378 25
327 379 9
366 371 25
380 292 20
367 326 21
381 365 25
370 378 7
366 367 8
382 383 6
361 354 16
321 384 13
324 256 8
355 385 11
386 387 22
371 379 17
388 381 27
383 301 23
255 336 13
345 359 13
376 316 7
353 373 21
375 389 16
305 373 30
390 391 28
376 2 24
255 336 15
347 348 20
255 275 22
392 393 29
372 394 27
395 396 20
397 390 17
272 365 24
272 388 24
389 352 5
398 399 22
297 2 21
400 272 22
385 379 29
255 275 30
393 382 7
401 386 25
380 339 27
402 403 14
254 404 19
345 359 20
405 406 29
406 395 30
378 339 7
407 408 26
409 376 25
332 314 29
322 410 17
310 384 17
178 309 24
411 376 25
405 395 16
337 379 16
365 412 13
337 379 13
413 414 12
415 416 18
350 410 20
380 339 19
377 336 28
417 407 25
319 400 26
359 418 25
297 2 14
216 400 23
419 410 26
395 396 21
420 335 29
421 422 9
348 337 18
421 422 13
332 314 22
423 424 12
284 356 22
336 356 15
425 347 16
309 426 22
397 427 14
391 2 7
324 248 11
396 374 10
428 429 15
345 310 18
430 413 8
431 432 26
409 411 18
332 314 29
433 434 29
423 424 20
435 418 23
436 248 5
401 437 6
397 427 28
384 435 15
438 418 17
255 323 30
322 419 9
325 248 28
374 375 17
439 417 10
324 392 24
412 440 27
415 441 27
409 2 9
297 2 23
442 433 19
431 432 13
421 422 14
443 444 21
425 347 20
437 387 26
337 366 16
445 248 17
255 275 17
337 366 5
380 369 10
424 421 11
420 335 6
418 446 16
399 447 8
416 441 5
324 415 25
448 402 27
397 2 26
449 450 15
451 443 11
360 452 23
397 2 28
453 216 16
454 450 18
443 444 9
450 332 26
452 455 26
397 423 28
216 440 25
194 445 27
409 2 18
425 379 17
380 339 10
450 442 10
404 456 6
432 455 10
451 444 28
357 394 18
434 457 13
447 458 29
452 431 6
314 457 15
255 283 24
443 444 30
427 459 28
456 460 8
438 345 15
422 297 18
409 2 24
457 380 20
324 436 15
275 283 21
428 429 15
461 449 24
462 463 11
460 461 6
397 464 17
465 448 24
466 467 11
397 425 24
449 450 17
467 379 13
468 2 29
373 2 13
469 470 7
457 471 24
438 449 14
397 427 22
324 441 24
178 472 28
387 473 29
178 426 24
414 454 22
420 410 18
426 439 28
397 464 29
441 451 24
453 440 8
339 324 22
420 410 13
453 440 12
453 440 13
441 451 20
444 398 7
324 403 19
397 379 7
450 457 23
420 410 15
449 474 16
397 379 29
446 428 11
450 475 7
441 451 18
420 410 10
414 450 12
464 466 17
476 468 6
463 409 14
446 428 10
477 478 25
420 410 11
474 430 9
438 449 12
438 252 22
397 379 21
472 408 14
455 394 21
478 462 18
459 405 28
479 480 12
252 460 20
471 324 13
481 2 9
397 379 16
482 483 30
324 248 19
438 429 17
450 475 14
450 457 11
397 2 19
484 477 16
457 324 10
438 449 29
357 452 27
483 478 19
324 403 5
324 485 29
438 449 16
397 2 13
397 379 11
449 450 13
450 457 27
403 469 27
324 441 15
324 485 22
458 486 28
487 488 9
478 462 18
470 484 29
481 2 12
450 457 24
356 324 28
475 481 12
450 457 15
324 465 18
394 489 21
473 449 27
450 457 19
490 357 9
450 255 24
324 178 23
401 473 25
324 403 21
487 488 17
438 449 20
324 491 19
440 487 29
458 486 13
1 492 6
429 449 28
1 492 6
1 449 16
478 462 25
1 493 17
488 479 8
477 478 7
492 494 13
397 476 23
438 401 14
457 324 8
486 495 16
485 478 25
408 496 17
324 403 26
479 397 24
1 492 9
490 248 19
487 479 15
450 457 12
487 479 17
497 453 19
449 450 18
379 2 19
498 478 10
420 410 26
1 492 10
489 248 7
477 498 6
482 483 27
480 397 18
499 420 29
491 478 11
495 482 22
493 500 8
477 478 19
494 499 5
449 450 29
248 478 30
324 490 21
1 438 20
494 499 26
487 479 21
410 497 21
477 478 10
496 478 23
500 449 8
1 449 20
248 478 26
1 438 17
