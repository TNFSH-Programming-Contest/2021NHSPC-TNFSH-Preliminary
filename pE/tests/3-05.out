19.5181133250147073616
